models/edr/dbt_artifacts/dbt_artifacts_state.sql

```diff
@@ -0,0 +1,20 @@
+{{
+  config(
+    materialized = 'view',
+    bind=False,
+    enabled=(target.type not in ['databricks', 'spark'])
+  )
+}}
+
+{% set hash_agg_expr %}
+md5({{ elementary.listagg("artifact_hash", order_by_clause="order by artifact_hash") }})
```

Comment:
<img width="1187" alt="image" src="https://user-images.githubusercontent.com/30181361/214549675-e599e598-3704-440e-aa7d-bf71df883b6e.png">

This is problematic in Redshift.
`65535 / 32 (MD5 Size) = 2048`
Meaning it'll fail on 2048+ model count.

I think I'll change the implementation to be `unique_id`, `artifact_hash` and then just diff them in Jinja.
What do you think? @haritamar

Reply:
Changed [here](https://github.com/elementary-data/dbt-data-reliability/pull/241/commits/656c8e6619bf028c08d591cdb1c221e2bd3514de).